concent_api/core/payments/mock.py

```diff
@@ -0,0 +1,52 @@
+
```

Comment:
Functions `is_provider_account_status_positive`, `tmp_is_provider_account_status_positive` and `make_forced_payment` defined at the bottom of `views.py` should be moved to payment backend (or use the ones from backend, if they are there alread) as described in implementation issue.